tools/restatectl/src/commands/cluster/config.rs

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2023 - 2025 Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+mod get;
+mod set;
+
+use std::fmt::{self, Display, Write};
+
+use cling::prelude::*;
+
+use restate_types::{
+    logs::metadata::DefaultProvider, partition_table::ReplicationStrategy,
+    protobuf::cluster::ClusterConfiguration,
+};
+
+// use crate::commands::cluster::overview::ClusterStatusOpts;
+
+#[derive(Run, Subcommand, Clone)]
+pub enum Config {
+    /// Print a brief overview of the cluster status (nodes, logs, partitions)
+    Get(get::ConfigGetOpts),
+    Set(set::ConfigSetOpts),
+}
+
+fn cluster_config_string(config: ClusterConfiguration) -> anyhow::Result<String> {
+    let mut w = String::default();
+
+    writeln!(w, "⚙️ Cluster Configuration")?;
+    write_leaf(
+        &mut w,
+        1,
+        false,
+        "Number of partitions",
+        config.num_partitions,
+    )?;
+    let strategy: ReplicationStrategy =
+        config.replication_strategy.unwrap_or_default().try_into()?;
+
+    write_leaf(&mut w, 1, false, "Replication strategy", strategy)?;
+
+    let provider: DefaultProvider = config.default_provider.unwrap_or_default().try_into()?;
+    write_default_provider(&mut w, 1, provider)?;
+
+    Ok(w)
+}
+
+fn write_default_provider<W: fmt::Write>(
+    w: &mut W,
+    depth: usize,
+    provider: DefaultProvider,
+) -> Result<(), fmt::Error> {
+    let title = "Provider";
+    match provider {
+        DefaultProvider::InMemory => {
```

Comment:
Are we building restatectl with the in memory and replicated loglet feature enabled?

Reply:
We are, but i will gate them anyway.